src/Salas_de_video/Pelicula.java

```diff
@@ -0,0 +1,50 @@
+package Salas_de_video;
+
+import java.util.ArrayList;
+
+public class Pelicula {
+    String nombre,duracion,descripcion;
+    ArrayList<String> generos;
+
+    public Pelicula(String nombre,String duracion,ArrayList<String> generos){
```

Comment:
```suggestion
    public Pelicula(String nombre,String duracion){
```

le añadimos generos por el metodo `añadirGenero`

Reply:
Añades JUnit test